docs/manual/external-tools.tex

```diff
@@ -767,20 +767,25 @@ \section{IntelliJ IDEA\label{intellij}}
   \<\$CHECKERFRAMEWORK/checker/dist/javac.jar>.
 
 \item
-  Add the annotated JDK library.
+  Add the annotated JDK library using the following method.
   Go to the ``Settings'' menu via (File>Settings) or (Ctrl-Alt-S) and
   navigate to the ``Java Compiler'' sub-menu via (Build, Execution,
   Deployment>Compiler>Java Compiler).  Add the following to the field
   ``Additional command line parameters'': \\
   \<-Xbootclasspath/p:\$CHECKERFRAMEWORK/checker/dist/jdk8.jar>
+  Note: It is a good idea to enclose the address in quotation
+  marks to escape special characters and spaces.
+  E.g. \<-Xbootclasspath/p:"\$CHECKERFRAMEWORK/checker/dist/jdk8.jar">
 
 \item
   Create an annotation profile and specify which checkers to run.
   Directly under the ``Java Compiler'' sub-menu is the ``Annotation
   Processors'' sub-menu. Navigate here and click on the green ``+'' to create
-  a new Annotation Processor profile. Select the modules you would like to
-  use checkers with from under the ``Default'' profile and move them to your
-  new profile via the button directly right of the ``-'' button.
+  a new Annotation Processor profile. Give this profile a unique name of your
+  choosing. Avoid using special characters or spaces. E.g. AnnotationProcessor.
```

Comment:
The sentence "E.g. AnnotationProcessor." is a ffragmentt.  Please combine these three sentences into one.  That will also clarify that "Select" is talking about a new task.

Reply:
Sure thing, will do.